website/content/docs/concepts/integrated-storage/autopilot.mdx

```diff
@@ -55,17 +55,33 @@ API](/api-docs/system/storage/raftautopilot#set-configuration) to manage its
 behavior. Autopilot gets initialized with the following default values.
 
 - `cleanup_dead_servers` - `false`
+  - This controls whether to remove dead servers from
+    the Raft peer list periodically or when a new server joins. This requires that
+    `min-quorum` is also set.
 
 - `dead_server_last_contact_threshold` - `24h`
+  - Limit on the amount of time
+    a server can go without leader contact before being considered failed. This
+    takes effect only when `cleanup_dead_servers` is set.
 
 - `min_quorum` - This doesn't default to anything and will need to be set to at
   least 3 when `cleanup_dead_servers` is set as `true`.
+  - Minimum number of servers allowed in a cluster before
+    autopilot can prune dead servers. This should at least be 3. Applicable only for
+    voting nodes.
 
 - `max_trailing_logs` - `1000`
+  - Amount of entries in the Raft Log that a server
+    can be behind before being considered unhealthy.
 
 - `last_contact_threshold` - `10s`
+  - Limit on the amount of time a server can go without leader contact before being considered unhealthy.
 
 - `server_stabilization_time` - `10s`
+  - Minimum amount of time a server must be in a stable, healthy state before it can become a voter. Until that happens, it will be visible as a peer in the cluster, but as a non-voter, meaning it won't contribute to quorum.
```

Comment:
```suggestion
  - Minimum amount of time a server must be in a healthy state before it can become a voter. Until that happens, it will be visible as a peer in the cluster, but as a non-voter, meaning it won't contribute to quorum.
```

IIRC it's just node health that impacts stabilization.

Reply:
Again, I was just bring the two pages in sync.

Reply:
Fixed in https://github.com/hashicorp/vault/pull/17397/commits/f43fc264e0168c3fefd8cf15009a7bc25c5be596